substrate/frame/revive/src/benchmarking/mod.rs

```diff
@@ -558,6 +558,24 @@ mod benchmarks {
 		assert_eq!(result.unwrap(), 1);
 	}
 
+	#[benchmark(pov_mode = Measured)]
+	fn seal_to_account_id() {
+		let len = <T::AccountId as MaxEncodedLen>::max_encoded_len();
+		let account = account::<T::AccountId>("target", 0, 0);
+		let address = T::AddressMapper::to_address(&account);
```

Comment:
Should be mapped here to incur worst case. Otherwise it short circuits into the fallback mapping.